test/integration/noderesourcetopology_cache_test.go

```diff
@@ -477,7 +481,13 @@ func TestTopologyCachePluginWithoutUpdates(t *testing.T) {
 			testCtx.Ctx, testCtx.CancelFn = context.WithCancel(context.Background())
 
 			cs := clientset.NewForConfigOrDie(globalKubeConfig)
-			extClient := versioned.NewForConfigOrDie(globalKubeConfig)
+			scheme := runtime.NewScheme()
+			utilruntime.Must(clientgoscheme.AddToScheme(scheme))
+			utilruntime.Must(topologyv1alpha2.AddToScheme(scheme))
```

Comment:
while this is probably OK, it seems wasteful to do each time. Can we do it once when test runs?

Reply:
the tests run independently, how can we init once and run cross all the tests? maybe we have to refactor to use a test.M?

Reply:
I sort of agree with @zwpaper, in the perspective to making each UT hermetic.